src/components/visualizations/utils/test/drilldownEventing.spec.tsx

```diff
@@ -339,9 +339,9 @@ describe("Drilldown Eventing", () => {
             jest.runAllTimers();
 
             expect(target.dispatchEvent.mock.calls[0][0].detail.drillContext.points).toEqual([
-                { intersection: [], x: 1, y: 100 },
```

Comment:
I see two functions changed but just one unit test extended. Is it correct? Aren't we missing some new tests?